dali/python/nvidia/dali/ops.py

```diff
@@ -45,6 +45,8 @@ def _docstring_generator(cls):
     ret = pre_doc
     ret += schema.Dox()
     ret += '\n'
+    if schema.AllowsSequences():
```

Comment:
There is still question if we want to add column to our operators table to indicate if given operator supports sequences.

Reply:
If you let me know how to do that I will take care of it in https://github.com/NVIDIA/DALI/pull/525

Reply:
You need to deal with https://github.com/NVIDIA/DALI/blob/master/docs/supported_op_devices.py.
Something like should do:
```
 formater = '{:{c}>{op_name_max_len}}  {:{c}>6}  {:{c}>6}  {:{c}>6}  {:{c}>7} {:{c}>9}\n'
    doc_table = ''
    doc_table += 'Below table lists all available operators and devices they can operate on.\n\n'
    doc_table += '.. |tick| image:: images/tick.gif\n'
    doc_table += formater.format('', '', '', '', '', op_name_max_len = op_name_max_len, c='=')
    doc_table += formater.format('Operator name', 'CPU', 'GPU', 'Mixed', 'Support', 'Sequences', op_name_max_len = op_name_max_len, c=' ')
    doc_table += formater.format('', '', '', '', '', op_name_max_len = op_name_max_len, c='=')
    for op in sorted(all_ops):
        is_cpu = '|tick|' if op in cpu_ops else ''
        is_gpu = '|tick|' if op in gpu_ops else ''
        is_mixed = '|tick|' if op in mix_ops else ''
        is_support = '|tick|' if op in support_ops else ''
        supports_seq = '|tick|' if b.GetSchema(op).AllowsSequences() else ''
op_doc = formater.format(op, is_cpu, is_gpu, is_mixed, is_support, supports_seq, op_name_max_len = op_name_max_len, c=' ')
```

Reply:
Will do (in #525)